.github/workflows/invoke-load-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Invoke Load Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      tests:
+        description: >
+          Array of test names
+          Example-["mysql_select", "mysql_insert"]
```

Comment:
We just decided to change the description to,

```
List of test names

This needs to be filled only if you want to run a specific set of tests. Example: [“foo”, “bar”]
```

Reply:
Fixed in 1f1ce36